centos-pypa-rust-aarch64/Dockerfile

```
@@ -0,0 +1,26 @@
+FROM quay.io/pypa/manylinux2014_aarch64
+
+WORKDIR /usr/src/app
+
+# Download rustup and checksum
+RUN curl https://static.rust-lang.org/rustup/dist/aarch64-unknown-linux-gnu/rustup-init.sha256 | awk '{print $1 " rustup-init"}' > checksum.txt && \
+    curl -O https://static.rust-lang.org/rustup/dist/x86_64-unknown-linux-gnu/rustup-init
```

```suggestion
    curl -O https://static.rust-lang.org/rustup/dist/aarch64-unknown-linux-gnu/rustup-init
```
Probably want the aarch version actually